src/main/utils/I3IpcHelper.js

```diff
@@ -0,0 +1,40 @@
+import { exec } from 'child_process';
+
+class I3IpcHelper {
+
+  isI3() {
+    return WindowManager.getWindowManagerName() === 'i3';
+  }
+
+  _hasI3Msg(callback) {
+    if(this.isI3()) {
+      exec('command -v i3-msg', error => {
```

Comment:
Pretty sure error needs brackets around it to meet the linting spec, if not linting then consistency :+1: